apps/web/app/css/global.css

```
@@ -10,6 +10,14 @@ html {
   height: 100%;
 }
 
+body:has(div[data-radix-popper-content-wrapper]) {
+  @media only screen and (min-width: 768px) {
+    @media only screen and (min-height: 100vh) {
+      overflow-y: scroll !important;
+    } 
+  } 
```

I think that media query can be combined into a single expression:

```suggestion
  @media only screen and (min-width: 768px) and (min-height: 100vh) {
    overflow-y: scroll !important;
  } 
```